core/src/main/resources/lib/form/optionalBlock.jelly

```diff
@@ -66,20 +66,22 @@ THE SOFTWARE.
 
   <j:choose>
     <j:when test="${attrs.title!=null}">
-      <tr class="optional-block-start row-group-start ${attrs.inline?'':'row-set-start'}" hasHelp="${attrs.help!=null}"><!-- this ID marks the beginning -->
-        <td colspan="3">
+      <div class="tr optional-block-start row-group-start ${attrs.inline?'':'row-set-start'}" hasHelp="${attrs.help!=null}"><!-- this ID marks the beginning -->
+        <div colspan="3">
           <f:checkbox name="${attrs.name}" class="optional-block-control block-control" onclick="javascript:updateOptionalBlock(this,true)"
                       negative="${attrs.negative}" checked="${attrs.checked}" field="${attrs.field}" title="${title}" />
-        </td>
-        <f:helpLink url="${attrs.help}" featureName="${title}"/>
-      </tr>
+          <f:helpLink url="${attrs.help}" featureName="${title}"/>
+        </div>
+      </div>
       <j:if test="${attrs.help!=null}">
         <f:helpArea />
       </j:if>
-      <tr class="rowvg-start" />
-      <d:invokeBody />
+      <div class="rowvg-start tr" />
+      <div class="container tr">
+        <d:invokeBody />
+      </div>
       <!-- end marker -->
-      <tr class="${attrs.inline?'':'row-set-end'} rowvg-end optional-block-end row-group-end" />
+      <div class="tr ${attrs.inline?'':'row-set-end'} rowvg-end optional-block-end row-group-end" />
```

Comment:
div tags are not self-closing, I think this should be `<div class="tr ${attrs.inline?'':'row-set-end'} rowvg-end optional-block-end row-group-end"></div>`